src/floatingobject.cpp

```diff
@@ -374,8 +374,20 @@ bool FloatingPositioner::CalcDrawingYRel(
     if (horizOverlapingBBox == NULL) {
         // Apply element margin and enforce minimal staff distance
         int staffIndex = staffAlignment->GetStaff()->GetN();
-        int minStaffDistance
-            = doc->GetStaffDistance(m_object->GetClassId(), staffIndex, m_place) * doc->GetDrawingUnit(staffSize);
+
+        int minStaffDistance = 0.0;
+        data_MEASUREMENTSIGNED minStaffDistanceMesurement
```

Comment:
```suggestion
        data_MEASUREMENTSIGNED minStaffDistanceMeasurement
```

-- will probably need to change the other instances of this too....